conduit/plugins/importers/algod/algod_importer_test.go

```diff
@@ -149,7 +151,8 @@ func TestInitCatchup(t *testing.T) {
 		targetRound sdk.Round
 		adminToken  string // to trigger fast-catchup
 		algodServer *httptest.Server
-		err         string
+		errInit     string
+		errGetGen   string
```

Comment:
It looks like you didn't end up using this?

Reply:
See for example lines 195/196. Sometimes I omitted them when they were `""`, but if that's confusing I can add these back in.

Reply:
It looks like 195 is also `""`

Reply:
The latest commit adds in all zero-valued `errInit` and `errGetGen`:

b8fef8fdb1a4936b8039361cbab6bc8456a99536

Reply:
I mean that none of these tests define `errGetGen`, so it could be removed.

Reply:
Thanks for these observations.  My gut was telling me that we need a failure case for full coverage and I half-baked a solution by providing `errGetGen`. But both of you observed that I didn't actually finish baking the cake. Now it should be all baked: 5b114a031eea414460009970334a2b8a42047355